Content.Server/DeviceLinking/Systems/PressurePlateSystem.cs

```diff
@@ -0,0 +1,114 @@
+using Content.Server.DeviceLinking.Components;
+using Content.Shared.DeviceLinking;
+using Robust.Shared.Physics.Components;
+using Content.Server.Storage.Components;
+
+using Robust.Shared.Physics.Events;
+using Content.Shared.StepTrigger.Systems;
+
+namespace Content.Server.DeviceLinking.Systems
+{
+    public sealed class PressurePlateSystem : EntitySystem
+    {
+        [Dependency] private readonly DeviceLinkSystem _signalSystem = default!;
+        [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+        [Dependency] private readonly SharedAudioSystem _audio = default!;
+
+        public override void Initialize()
+        {
+            base.Initialize();
+            SubscribeLocalEvent<PressurePlateComponent, ComponentInit>(OnInit);
+            SubscribeLocalEvent<PressurePlateComponent, StepTriggerAttemptEvent>(HandleTriggerAttempt);
+            SubscribeLocalEvent<PressurePlateComponent, EndCollideEvent>(OnEndCollide);
+            SubscribeLocalEvent<PressurePlateComponent, StartCollideEvent>(OnStartCollide);
+        }
+
+        private void OnInit(EntityUid uid, PressurePlateComponent component, ComponentInit args)
+        {
+            _signalSystem.EnsureSourcePorts(uid, component.PressedSignal, component.ReleasedSignal);
+        }
+
+        private void HandleTriggerAttempt(EntityUid uid, PressurePlateComponent component, ref StepTriggerAttemptEvent args)
```

Comment:
why is this using step trigger

Reply:
without it, the collision is not processed

Reply:
so use itemplacer and its not needed